### ISSUE TITLE
Screen Reader: Global: Toggle/checkbox label announced twice, as text and programmatic label

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77497]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Preferences button and double-tap to activate it
4. Swipe right to focus the toggle buttons and note the focus behavior of the toggle buttons and their adjacent texts

Global issue for all toggle buttons and checkboxes

## Expected Result:
Only the toggle buttons should receive focus.
The visible text next to the toggle should not be focusable and should be hidden from assistive technology.
Screen readers announce the label once via the toggle, for example:
"Receive relevant feature updates and Expensify news, switch, on".

## Actual Result:
The visible text is focusable separately, resulting in redundant announcements:
The text label is announced as "Receive relevant feature updates and Expensify news", and then;
The toggle itself is announced as "Receive relevant feature updates and Expensify news, switch, on".

This creates duplicate unnecessary announcement for screen reader users.

## Area issue was found in:
Preferences and  Subsections

## Failed WCAG checkpoints
WCAG 1.3.2 Meaningful Sequence
https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence

## User impact:
A meaningful sequence that matches a logical reading order helps people who rely on assistive technologies that read content aloud. The meaning evident in the sequencing of the information in the default presentation will be the same when the content is presented in spoken form.

## Suggested resolution:
Hide the visible text from screen readers using accessible={false}, since it is already included in the toggle’s accessibilityLabel.
This way, only the toggle switch receives focus, preventing redundant announcements.
https://appt.org/en/docs/react-native/samples/accessibility-focusability
https://reactnative.dev/docs/accessibility#accessible

## Environment(s)
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
<img width="1080" height="2195" alt="Image" src="https://github.com/user-attachments/assets/415627ab-e258-43cc-971d-7df2dbf301eb" />

https://github.com/user-attachments/assets/6dab8900-ebae-4f37-8387-a668e58d9417

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7008154
**Issue reported by:** NA
**Slack conversation:** NA
